ktor-server/ktor-server-core/jvm/src/io/ktor/features/CallLogging.kt

```diff
@@ -221,7 +223,34 @@ private class MDCSurvivalElement(mdc: Map<String, String>) : ThreadContextElemen
     private object Key : CoroutineContext.Key<MDCSurvivalElement>
 }
 
-private fun defaultFormat(call: ApplicationCall): String = when (val status = call.response.status() ?: "Unhandled") {
-    HttpStatusCode.Found -> "$status: ${call.request.toLogString()} -> ${call.response.headers[HttpHeaders.Location]}"
-    else -> "$status: ${call.request.toLogString()}"
+private fun defaultFormat(call: ApplicationCall): String {
+    if (!AnsiConsole.isInstalled()) {
+        AnsiConsole.systemInstall()
```

Comment:
In some cases, this method can throw an exception. Could we fall back to the regular log in such cases? 
It could be part of the `colored` method for instance

Reply:
We also think about flag to opt-out the feature